src/app/clusters/temperature-measurement-server/temperature-measurement-server.cpp

```diff
@@ -15,78 +15,95 @@
+    if (EMBER_ZCL_STATUS_SUCCESS != status)
+    {
+        (void) emberAfTemperatureMeasurementClusterSetMeasuredValueCallback(endpoint, 0);
+    }
+
+    status = emberAfTemperatureMeasurementClusterGetMinMeasuredValue(endpoint, &value);
+
+    if (EMBER_ZCL_STATUS_SUCCESS != status)
+    {
+        (void) emberAfTemperatureMeasurementClusterSetMinMeasuredValueCallback(endpoint, INT16_MIN);
+    }
+
+    status = emberAfTemperatureMeasurementClusterGetMaxMeasuredValue(endpoint, &value);
+
+    if (EMBER_ZCL_STATUS_SUCCESS != status)
+    {
+        (void) emberAfTemperatureMeasurementClusterSetMaxMeasuredValueCallback(endpoint, INT16_MIN);
+    }
+}
+
+EmberAfStatus emberAfTemperatureMeasurementClusterSetMeasuredValueCallback(chip::EndpointId endpoint, int16_t measuredValue)
 {
-    EmberAfStatus status;
-    // FIXME Use real values for the temperature sensor polling the sensor using the
-    //       EMBER_AF_PLUGIN_TEMPERATURE_MEASUREMENT_SERVER_MAX_MEASUREMENT_FREQUENCY_S macro
-    EndpointId endpointId = 1; // Hardcoded to 1 for now
-    int16_t newValue      = 0x1234;
-
-    status = emberAfWriteAttribute(endpointId, ZCL_TEMP_MEASUREMENT_CLUSTER_ID, ZCL_CURRENT_TEMPERATURE_ATTRIBUTE_ID,
-                                   CLUSTER_MASK_SERVER, (uint8_t *) &newValue, ZCL_INT16S_ATTRIBUTE_TYPE);
-    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    EmberAfStatus status = emberAfWriteAttribute(endpoint, ZCL_TEMP_MEASUREMENT_CLUSTER_ID, ZCL_TEMP_MEASURED_VALUE_ATTRIBUTE_ID,
+                                                 CLUSTER_MASK_SERVER, (uint8_t *) &measuredValue, ZCL_INT16S_ATTRIBUTE_TYPE);
+    if (EMBER_ZCL_STATUS_SUCCESS != status)
     {
-        emberAfTempMeasurementClusterPrint("Err: writing temperature: %x", status);
-        return;
+        // emberAfTemperatureMeasurementClusterPrintln("ERR: writing present value %x", status);
     }
+
+    return status;
 }
 
-EmberAfStatus emberAfPluginTemperatureMeasurementSetValueCallback(EndpointId endpoint, int16_t value)
+EmberAfStatus emberAfTemperatureMeasurementClusterSetMinMeasuredValueCallback(chip::EndpointId endpoint, int16_t minMeasuredValue)
 {
-    EmberAfStatus status = emberAfWriteAttribute(endpoint, ZCL_TEMP_MEASUREMENT_CLUSTER_ID, ZCL_CURRENT_TEMPERATURE_ATTRIBUTE_ID,
-                                                 CLUSTER_MASK_SERVER, (uint8_t *) &value, ZCL_INT16S_ATTRIBUTE_TYPE);
-    if (status != EMBER_ZCL_STATUS_SUCCESS)
+    EmberAfStatus status =
+        emberAfWriteAttribute(endpoint, ZCL_TEMP_MEASUREMENT_CLUSTER_ID, ZCL_TEMP_MIN_MEASURED_VALUE_ATTRIBUTE_ID,
+                              CLUSTER_MASK_SERVER, (uint8_t *) &minMeasuredValue, ZCL_INT16S_ATTRIBUTE_TYPE);
+    if (EMBER_ZCL_STATUS_SUCCESS != status)
     {
-        emberAfTempMeasurementClusterPrint("Err: writing temperature: %x", status);
+        // emberAfTemperatureMeasurementClusterPrintln("ERR: writing present value %x", status);
     }
+
+    return status;
```

Comment:
`return emberAfWriteServerAttribute(endpoint, ZCL_TEMP_MEASUREMENT_CLUSTER_ID, ZCL_TEMP_MIN_MEASURED_VALUE_ATTRIBUTE_ID, (uint8_t *) &minMeasuredValue, ZCL_INT16S_ATTRIBUTE_TYPE);`

Use `emberAfWriteServerAttribute` instead of `emberAfWriteAttribute` and let the caller log something is the return status is different than `EMBER_SUCCESS`.